### DISCUSSION TITLE
Accessing TPOT dashboard behind a reverse proxy

### DISCUSSION THREAD

**Discussion by mayank-joshi-01** [telekom-security/tpotce#1555] [Q&A]
I am trying to access TPOT reverse proxy dashboard from another nginx reverse proxy, but I get 502 error code, I think the problem is with ssl, what are the steps to bring TPOT nginx from https to http? I have edited tpotce/docker/nginx/dist/conf/tpotweb.conf to list on http, and after I did docker compose -f tpotce/docker-compose.yml down && docker compose -f tpotce/docker-compose.yml up -d --build, but it did not change to http.

**Comment by t3chn0m4g3** [comment]
Check out the Wiki for an example, you also need to add a `volume` for the changed config for the nginx service in the `docker-compose.yml`.

**Comment by mayank-joshi-01** [comment]
I changed the tpotweb.conf and build a custom local image out of it and used it into the docker-compose.yml, its working fine now.

Thanks for your time.
